config/config.go

```diff
@@ -161,22 +179,30 @@ func Load(filename string, loadFromCli ...bool) (*Config, error) {
 		config.FrontendPath = "/app"
 	}
 	if len(config.TlsCert) < 1 {
-		config.TlsCert = path.Join(opts.Opts.BaseDir, "server.crt")
+		if defaults.TlsCert != "" {
```

Comment:
this is not possible currently, why having this code?

Reply:
I wanted all params to be reflected in defaults as it would be expected because of the type of"defaults" variable. But i think you're right, i'll remove unused checks